db/WiredTiger.turtle

```diff
@@ -0,0 +1,6 @@
+WiredTiger version string
```

Comment:
**TODO FOR LOST POINTS**

For some reason, your ``db`` folder did not get ``gitignore'd`` properly. Please fix.